spec/controllers/idv/by_mail/request_letter_controller_spec.rb

```diff
@@ -261,6 +261,6 @@ def expect_resend_letter_to_send_letter_and_redirect(otp:)
     expect(gpo_confirmation_maker).to receive(:perform)
     expect(gpo_confirmation_maker).to receive(:otp) if otp
     expect { put :create }.to change { ActionMailer::Base.deliveries.count }.by(1)
-    expect(response).to redirect_to idv_come_back_later_path
+    expect(response).to redirect_to idv_gpo_letter_enqueued_path
```

Comment:
Can we leave this as `idv_letter_enqueued_path`, or maybe `idv_by_mail_letter_enqueued_path` ?